index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobile First Project</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" class="logo">My Logo</a>
+      <nav class="hamburger">
+        <span class="navbar"></span>
+        <span class="navbar"></span>
+        <span class="navbar"></span>
+      </nav>
+    </header>
+
+    <section class="main-container">
+      <div class="title">
+        <h1>This is Mert Glad to see you here!</h1>
+      </div>
+      <dev class="intro">
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <p class="connect">LET'S CONNECT</p>
```

Comment:
- I think it is important to wrap your `<p class="connect">LET'S CONNECT</p>` in an anchor tag. The `a` tag will let you connect to other sections of the website, using href, or maybe a link to your resume when you get one. You can do something like this 👉🏽 `<p class="connect"><a href="#">LET'S CONNECT</a></p> `😄

Reply:
Fixed it. Thanks. However, I can't get rid of underlining for the writing. Although I used text-decoration: none. It worked for the logo part but not for this part. How do I do it?